sdk/data/aztables/README.md

```diff
@@ -58,6 +60,14 @@ To use an account [shared key][azure_shared_key] (aka account key or access key)
 az storage account keys list -g MyResourceGroup -n MyStorageAccount
 ```
 
+```golang
+cred, err := aztables.NewSharedKeyCredential("<accountName>", "<accountKey>")
+handle(err)
+client, err := aztables.NewServiceClientWithSharedKey(serviceURL, cred, nil)
+handle(err)
+```
+
+##### Creating the client with an AAD credential
```

Comment:
I believe we want to place the ADD cred before the shared key as we try to promote AAD over shared key.

Reply:
Fixed